example/index.html

```diff
@@ -105,6 +105,10 @@
                  viewer.next();
              });
 
+            // Debug/example code: Listen for tile source requests and loads
+            viewer.on('detail-tile-source-requested', function(e) { console.log('detail tile source requested', e.detail)});
+            viewer.on('detail-tile-source-opened', function(e) { console.log('detail tile source opened', e.detail)});
+
```

Comment:
Yep. This is the kind of pattern I was thinking of. Good to pass in the global dispatcher (below) as well. Thanks for setting this up. I think I can set up an example page for the layering interface using this kind of API (will use some imaginary event names for now).